game/hud/src/services/session/layout.ts

```diff
@@ -109,18 +121,80 @@ const initialState = {
   widgets: defaultWidgets(),
 }
 
-let getInitialState = () => {
-  var storedState = JSON.parse(localStorage.getItem(localStorageKey)) as LayoutState;
+interface Size {
+  width: number;
+  height: number;
+}
+
+const px2pcnt = (pos: Position, screen: Size) : Position => {
+  // converts pixel co-ordinates into % of display size
+  return {
+    x: pos.x / screen.width,
+    y: pos.y / screen.height,
+    width: pos.width,           // don't scale width/height
+    height: pos.height,
+    scale: 1
```

Comment:
yes it can, you are wrong.  it is a private function and the module knows what is calling it